internal/db/db.go

```diff
@@ -24,9 +24,11 @@ func InitDB() {
 	dbPass := os.Getenv("DB_PASSWORD")
 	dbName := os.Getenv("DB_NAME")
 
-	db, err := sql.Open("postgres", fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=%s sslmode=disable", dbHost, dbUser, dbPass, dbName, dbPort))
+	var connErr error
 
-	if err != nil {
+	db, connErr = sql.Open("postgres", fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=%s sslmode=disable", dbHost, dbUser, dbPass, dbName, dbPort))
+
+	if connErr != nil {
 		panic(err.Error())
```

Comment:
Also, your internal packages should not panic or call `log.Fatal`... they should return the errors back to main which will decide what to do with them... Line 18 calls `log.Fatal`

Reply:
I'll make the change